### PR TITLE
Update the ecosystem model section of the docs

### PR DESCRIPTION
**Pull request by waldirmontoya25**
## Hey, I just made a Pull Request!

- [x] Added or updated documentation

### CODE REVIEW

File: docs/features/software-catalog/system-model.md [hunk 1]
```diff
@@ -108,3 +108,21 @@ domain would come with some documentation on how to accept payments for a new
...
+belong to Group entities in the catalog.
+
+### Group
+
+A group describes an organizational entity, such as for example a team, a
+business unit, or a loose collection of people in an interest group.
+
+### Types
+
+The type field in the system has no set meaning. It is up to the user to assign their own types and use them as desired, such as for link validation or creating custom UI components. Some common pre-defined types are depicted in the ecosystem modeling diagram."
```
Comment (kuangp) [inline docs/features/software-catalog/system-model.md:124]:
```suggestion
The type field in the system has no set meaning. It is up to the user to assign their own types and use them as desired, such as for link validation or creating custom UI components. Some common pre-defined types are depicted in the ecosystem modeling diagram.
```

File: docs/features/software-catalog/system-model.md [hunk 2]
```diff
@@ -108,3 +108,26 @@ domain would come with some documentation on how to accept payments for a new
...
+### Group
+
+A group describes an organizational entity, such as for example a team, a
+business unit, or a loose collection of people in an interest group.
+
+### Location
+
+A location is a marker that references other places to look for catalog data.
+
+### Type
```
Comment (freben) [inline docs/features/software-catalog/system-model.md:125]: Only thing I note here is the top-level heading structure. Note that we currently have `## Core Entities` and `## Ecosystem Modeling`. Maybe User and Group could go under a new `## Organizational Entities`, and then ... hm, type and template could be under a `## Other` 🤷

**Review by freben — COMMENTED** [review]
Great stuff! Just one comment

File: docs/features/software-catalog/system-model.md (hunk 2 above)
Comment (waldirmontoya25) [reply docs/features/software-catalog/system-model.md:125]: The intention was to show a unified view of how the different pieces fit together, making it easier for a new adopter to understand the relationships between the objects (Ecosystem Modeling). 

What do you think about adding the sections you mention but also a graph that unifies all the pieces?

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Thanks @waldirmontoya25! :pray:

**Pull request merged**